TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -298,9 +298,18 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		_mediaBrowserView: null,
 
 		_beforeMediaBrowserIsShown: function () {
-			var that = this;
+			var that = this,
+				value = null;
+
+			try {
+				value = JSON.parse(this.get('value'));
+			} catch (exception) {}
```

Comment:
is this really necessary? why would it ever be invalid json? isn't there another way to check if it should be parsed or not?